setup.py

```diff
@@ -15,7 +15,7 @@
     ],
     'lint': [
         "flake8==3.4.1",
-        "isort>=4.2.15,<5",
+        "isort==4.3.4",
```

Comment:
A newer `isort` version rejects the trailing comma for multi line imports and the advertised `--trailing-comma` doesn't seem to work either. Pinning to the last version that works for now.